src/FileFormatConverter.h

```diff
@@ -0,0 +1,322 @@
+/*
```

Comment:
Would this file be better in the profilerlib directory, since it seems to only be useful within the profilerlib code?